Bottom: 170e1bd9f4bd26b57fbe9ec6d33684d47e1ad087
Top:    81826e534fbc052d88bf84eb0a6e91177c0d6d8e
Author: Jason Wang <jasowang@redhat.com>
Date:   2016-05-24 16:58:53 +0800

Refresh of vhost-device-iotlb-api

---

diff --git a/include/uapi/linux/vhost.h b/include/uapi/linux/vhost.h
index 78f7d8f..f2e5757 100644
--- a/include/uapi/linux/vhost.h
+++ b/include/uapi/linux/vhost.h
@@ -47,6 +47,7 @@ struct vhost_vring_addr {
 	__u64 log_guest_addr;
 };
 
+/* no alignment requirement */
 struct vhost_iotlb_msg {
   __u64 iova;
   __u64 size;
